shopinvader_product_brand/views/shopinvader_brand_view.xml

```diff
@@ -1,120 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-<odoo>
-
-<record id="view_shopinvader_brand_form" model="ir.ui.view">
-    <field name="model">shopinvader.brand</field>
-    <field name="arch" type="xml">
-        <form>
-            <group name="top">
-                <field name="backend_id" />
-                <field name="lang_id" />
-                <field name="active" />
-            </group>
-            <notebook>
-                <page name="description" string="Description">
-                    <group name="description">
-                        <separator string="Short Description" colspan="2" />
-                        <field name="short_description" nolabel="1" colspan="2" />
-                        <separator string="Description" colspan="2" />
-                        <field name="description" nolabel="1" colspan="2" />
-                    </group>
-                </page>
-                <page name="seo" string="SEO">
-                    <group name="meta_and_url">
```

Comment:
Are all the `url` fields still needed on a `product.brand` ? And should they be present in view?

Reply:
Removing all these fields from views. Can be added by whoever needs them.